GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -145,6 +146,66 @@ const float* Mesh::getBoundingBoxInfo() {
     return bounding_box_info_;
 }
 
+float* Mesh::getTransformedBoundingBoxInfo(glm::mat4 *Mat) {
+
+    if (have_bounding_box_ == false)
+        getBoundingBoxInfo();
+
+    glm::mat4 M = *Mat;
+    float a, b;
+    float *transformed_bounding_box = new float[6];
+
```

Comment:
But allocating one on the stack and returning is not the most correct way to do it. Once the method returns, there is no guarantee that the contents would exist as the stack would get cleared out. Or do you mean allocating one on the stack in scene_object.cpp and passing it over?